metacat-common-server/src/main/java/com/netflix/metacat/common/server/util/PoolStatsInterceptor.java

```diff
@@ -38,10 +37,10 @@
      * Metric name.
      */
     public static final String PROP_METRIC_NAME = "name";
-    private Gauge metricNameTotalGauage;
-    private Gauge metricNameActiveGauage;
-    private Gauge metricNameIdleGauage;
-    private final Registry registry = Spectator.globalRegistry();
+    private AtomicInteger metricNameTotalGauge;
+    private AtomicInteger metricNameActiveGauge;
+    private AtomicInteger metricNameIdleGauge;
+    private final MeterRegistry registry = io.micrometer.core.instrument.Metrics.globalRegistry;
```

Comment:
I think this class can't inject the registry and the global.Registry() was used. Try if we can use Metrics.globalRegistry instead of including the package path.